versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.0.0, < 5.0"
+      version = ">= 4.1"
```

Comment:
```suggestion
      version = ">= 4.0.0, < 6"
```

Reply:
`>= 4.1` was probably intended for https://github.com/GoogleCloudPlatform/terraform-google-secret-manager/blob/main/test/setup/versions.tf#L22